script/upload_to_rpm_repo

```diff
@@ -14,80 +14,79 @@ set -exu
 #
 # Date          Channel       Fingerprint           Full Key
 #
+# NOT ACTIVE    Release       0xA8580BDC82D3DC6C    9BD198A2D848C482E3550697A8580BDC82D3DC6C
 # 06/26/2019    Release       0x4FE13824E3FFC656    E85FFA8E2E90B40B33ED39274FE13824E3FFC656
 # 12/29/2018    Beta          0x0B31DBA06A8A26F9    9228DBCE20DDE5EC46488DE90B31DBA06A8A26F9
 # 12/29/2018    Dev           0x0B31DBA06A8A26F9    9228DBCE20DDE5EC46488DE90B31DBA06A8A26F9
 # 12/29/2018    Nightly       0x0B31DBA06A8A26F9    9228DBCE20DDE5EC46488DE90B31DBA06A8A26F9
 
-BUCKET=${1:-brave-browser-rpm-staging-nightly}
-GPG_KEY_ID=${2:-D8BAD4DE7EE17AF52A834B2D0BB75829C2D4E821} # Defaults to New Brave Software signing key
-GPG_KEY_SHORT_ID=${GPG_KEY_ID:(-8)}
+S3_BUCKET=${1}
+GPG_KEY_ID=${2}
+TMP_REPO=$(mktemp -d)
+# SNAP_TAG=$(date +'%s')
 
-if [ -z "${CHANNEL}" ]; then
-    echo "Error: Environment variable CHANNEL not set! Exitting..."
+if [ -n "$S3_BUCKET" ]; then
+    echo "Error: Please pass the S3 bucket argument as the first argument"
     exit 1
 fi
 
-if [[ $2 == 0x* ]]; then
-    echo "Error: Please pass the Full GPG KEY ID to sign with! (i.e. 'E85FFA8E2E90B40B33ED39274FE13824E3FFC656')"
+if [ -n "$GPG_KEY_ID" ]; then
+    echo "Error: Please pass the full GPG Key ID as the second argument"
     exit 1
 fi
 
-if [ "$GPG_KEY_ID" = "E85FFA8E2E90B40B33ED39274FE13824E3FFC656" ]; then
+case "$GPG_KEY_ID" in
+    "D8BAD4DE7EE17AF52A834B2D0BB75829C2D4E821"|"9BD198A2D848C482E3550697A8580BDC82D3DC6C")  # Release keys
```

Comment:
Here too, I think it should be `...656` and `...C6C`.

Reply:
Fixed by adding all 3 key identifiers.